agent/php_agent.h

```diff
@@ -902,6 +902,22 @@ extern const char* nr_php_zend_execute_data_scope_name(
  */
 extern uint32_t nr_php_zend_execute_data_lineno(
     const zend_execute_data* execute_data);
+
+/*
+ * Purpose : Return a uint32_t (zend_uint) line number value of zend_function.
+ *
+ * Params  : 1. zend_function.
+ *
+ * Returns : uint32_t lineno value
+ *
+ */
+static inline uint32_t nr_php_zend_function_lineno(const zend_function* func) {
+  if (NULL != func) {
+    return func->op_array.line_start;
```

Comment:
Should `func` be checked if it is a user function? Do all types of `zend_function` that may be passed to `nr_php_zend_function_lineno` have `line_start` defined/initialized? Is `nr_php_zend_execute_data_lineno` still needed?

Reply:
Thanks, addressed in [3a3a44f](https://github.com/newrelic/newrelic-php-agent/pull/558/commits/3a3a44f38797a2900577f65ef385ab46cbd35e56)